recipes/shove/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "shove" %}
+{% set version = "0.6.6" %}
+{% set sha256 = "f1c643b5c68f45ad3a3fb6f39aa7f1a9221d0c387a135df06123990f83a734ec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/f5/c6/75b6d46b76b43626d7da946331827a7ccb38f2b4683808c74f3eff2519a5/shove-0.6.6.tar.bz2#md5=1f4f3acef63f2675643aa6abb7c7547d
```

Comment:
Seems to not be able to find 0.6.6.tar.bz2 on pypi.io. Which is really strange since the web front end clearly lists it. I was able to download the zip file though. I was able to get past the download issue by using the zip file instead. Revert this last commit that adds the ugly url and change the extension to .zip here on the `url` and also on the `fn`. The sha256 for that zip is `ec5034f9abf809b1e24cbf28e475a6339c31fa9459940e936974a4a669c094a5`.